app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TabWebContentsDelegate.java

```diff
@@ -5,14 +5,17 @@
 
 import com.igalia.wolvic.browser.api.WSession;
 
+import org.chromium.base.task.PostTask;
+import org.chromium.base.task.TaskTraits;
 import org.chromium.blink.mojom.DisplayMode;
-import org.chromium.components.embedder_support.delegate.WebContentsDelegateAndroid;
 import org.chromium.content_public.browser.InvalidateTypes;
 import org.chromium.content_public.browser.LoadUrlParams;
 import org.chromium.content_public.browser.WebContents;
+import org.chromium.content_public.common.ResourceRequestBody;
```

Comment:
Is this used anywhere?

Reply:
Seems to be a leftover from previous attempts. Will remove it.